src/postgraphile/cli.ts

```diff
@@ -572,7 +572,14 @@ const coerce = (o: ReturnType<typeof parsePgConnectionString>): PoolConfig => {
   return {
     ...o,
     application_name: o['application_name'] || undefined,
-    ssl: o.ssl != null ? !!o.ssl : undefined,
+    ssl:
+      o.ssl == null
+        ? undefined
+        : o.ssl.rejectUnauthorized == null
+        ? !!o.ssl
+        : {
+            rejectUnauthorized: !!o.ssl.rejectUnauthorized,
+          },
```

Comment:
Maybe we should pass it through unmodified if it's an object?

```suggestion
    ssl:
      o.ssl == null
        ? undefined
        : typeof o.ssl === 'object' && !Array.isArray(o.ssl)
        ? o.ssl
        : !!o.ssl,
```

Reply:
I think it depends on why `coerce` exists in the first place. Seems like it doesn't really do anything in most cases i.e. when `pg-connection-string` is well-behaved?

Passing the object as-is would obviously be more "future-proof" but it is equivalent as of now since `rejectUnauthorized` is the only thing that can be returned in this object.

Feel free to make a decision either way, I have no preference.

Reply:
Any thoughts on the above? @benjie Happy to just commit your suggestion if we are fine with not validating the object

Reply:
Leave it be. It’s been a hectic week; sorry!

Reply:
I went in a slightly different direction; if `rejectUnauthorized` is set then we handle the whole object. It's a bit weird but should be backwards-compatible (at least with _intent_).